cfme/tests/infrastructure/test_vm_power_control.py

```diff
@@ -270,12 +293,12 @@ def test_start_from_suspend(
         if_scvmm_refresh_provider(test_vm.provider)
         test_vm.wait_for_vm_state_change(
             desired_state=test_vm.STATE_ON, timeout=720, from_details=True)
-        self._wait_for_last_boot_timestamp_refresh(test_vm, last_boot_time, timeout=600)
+        wait_for_last_boot_timestamp_refresh(test_vm, last_boot_time, timeout=600)
         soft_assert(
             test_vm.provider.mgmt.is_vm_running(test_vm.name), "vm not running")
 
 
-def test_no_template_power_control(provider, setup_provider_funcscope, soft_assert):
```

Comment:
Why removing this? if the test is run individually it will need the setup provider wont it?